Angular/src/app/login/login.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl,FormGroup, Validators } from '@angular/forms'
+import { HttphandlerService } from '../httphandler.service'
+import { Router } from '@angular/router';
+import { CookieService } from 'ngx-cookie-service';
+import privateKey from '../../../../privateKey.js'
```

Comment:
`import { privateKey } from '../../../../privateKey.js'` for consistency

Reply:
ok